core/src/main/java/hudson/search/SuggestedItem.java

```diff
@@ -62,7 +62,7 @@ private void getPath(StringBuilder buf) {
             buf.append(item.getSearchName());
         else {
             parent.getPath(buf);
-            buf.append(' ').append(item.getSearchName());
+            buf.append(" → ").append(item.getSearchName());
```

Comment:
While much better than before (no separator), 

```suggestion
            buf.append(" » ").append(item.getSearchName());
```

for consistency elsewhere in Jenkins (e.g., List View configuration with "Recurse in subfolders" enabled).

Additionally (but easily deferred to the future I think, it was bad before too), it's weird to show raw job names here, instead of display names.

Reply:
Done, thanks

Reply:
> for consistency elsewhere in Jenkins (e.g., List Views with "Recurse in subfolders" enabled).

Addressed in https://github.com/jenkinsci/jenkins/pull/7569/commits/7af9147cf9494093d12c5e75645d393f9bbd79dc

Reply:
Unresolving thread for

> Additionally (but easily deferred to the future I think, it was bad before too), it's weird to show raw job names here, instead of display names.

(It's fine to defer since I think it's not a regression.)